build.fsx

```diff
@@ -272,14 +272,27 @@ Target "CodeGen.NetCore" (fun _ ->
     let fsLex fsl out = runInDir (toolDir + "fslex.exe") "%s --unicode %s -o %s" fsl lexArgs out
     let fsYacc fsy out m o = runInDir (toolDir + "fsyacc.exe") "%s %s %s %s %s -o %s" fsy lexArgs yaccArgs m o out
 
+#if FABLE_COMPILER
```

Comment:
I know, it's there just until the new tooling get released and FCS is migrated to use it.